presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorSplitManager.java

```diff
@@ -15,10 +15,21 @@
 
 import com.facebook.presto.spi.ConnectorSession;
 import com.facebook.presto.spi.ConnectorSplitSource;
+import com.facebook.presto.spi.ConnectorTableHandle;
 import com.facebook.presto.spi.ConnectorTableLayoutHandle;
 
 public interface ConnectorSplitManager
 {
+    default ConnectorSplitSource getSplits(
```

Comment:
Didn't quite get the intuition behind this new interface. We don't really need this, do we?